packages/strapi-types/index.ts

```diff
@@ -18,17 +21,43 @@ type BaseModel = {
   };
 };
 
-type ComponentModel = BaseModel & {};
+export type Model = {
+  collectionName: string;
+  uid: string;
+};
+
+export type ComponentModel = BaseModel & {};
+
+type Entity = {
+  [key: string]: Model;
+};
+
+export type Plugin = {
+  [key: string]: any;
+};
 
-export type Strapi = {
+export class Strapi {
   components: {
     [key: string]: ComponentModel;
-  };
-  plugins: {
-    [key: string]: any;
-  };
+  } = {};
+  plugins: Plugin = {};
   contentTypes: {
     [key: string]: any;
-  };
-  query: (s: string) => any;
-};
+  } = {};
+  query = (s: string): Entity => ({
+    fakeModel: {
```

Comment:
?
create function which return correct value not fake, 

if somebody won't add fakeModel, and this function will be used, than test will fail

Reply:
fixed